dyn_em/solve_em.F

```diff
@@ -4537,6 +4539,29 @@ SUBROUTINE solve_em ( grid , config_flags  &
                             ipsy, ipey, jpsy, jpey, kpsy, kpey   )
 
 
+! Get domain clock
```

Comment:
@thomasedds 
Thomas,
We do not want to have a diagnostic called directly from the solve routine. This needs to be pushed down several layers: solve, after_all_rk_steps, diagnostic driver, then maybe your scheme.